source/_posts/2024-03-04-dashboard-chapter-1.markdown

```diff
@@ -0,0 +1,262 @@
+---
+layout: post
+title: "A Home-Approved Dashboard chapter 1: Drag-and-drop, Sections view, and a new grid system design!"
+description: "Wow! At long last!! The stars have aligned, and our experimental drag-and-drop feature for dashboards is finally here!"
+date: 2024-03-04 00:00:01
+date_formatted: "March 5, 2024"
```

Comment:
Pick if you want to publish this on the 5th or still make it today.

If it is the 5th, make sure you also rename the file.

Reply:
If you can help me push it, then 4th is great :)